standards/SPIFFE_Federation.md

```diff
@@ -307,6 +307,56 @@ As described in the [Trustworthiness of the Bundle Endpoint Server](#74-trustwor
+
+Operators should use caution and good judgement in evaluating if and how they should process claims asserted in SVIDs from foreign trust domains.
+
+It bears repeating that implementations should never process any claims from SVIDs that fail authentication (except in a forensic context).
+
+#### 7.7.1 SVID Claims
+
+For the purpose of this discussion, a *claim* is an assertion about some attribute the subject of an SVID, or about the SVID itself.  The canonical SPIFFE claim is the SPIFFE ID, which is a claim about the subject of the SVID.  An example of a claim about the SVID itself is the expiration time.
+
+SVID formats (to date including X.509 and JWT) afford flexibility to encode additional claims, including claims that have not been evaluated by any standards body.
+
+In considering how to process claims made by foreign trust domains, it is helpful to consider three categories of claims:
+
+  1. Claims required by and/or defined by the SPIFFE specifications for the SVID type
+  2. Claims that are standardized by the relevant standards body for the document format (i.e. ITU-T for X.509, IETF for JWT)
+  3. Non-standardized claims
+
+Claims in the first category are well-defined in their meaning, and in many cases implementations are required to process them to adhere to SPIFFE specifications.
+
+Claims in the second category have standardized meaning, but operators have some choice whether to and how to process them.
+
+Claims in the third category are not standarized, so operators need to consider them on a case-by-case basis, possibly including navigating different meanings by different trust domains.  Needless to say, this category should be approached with the highest degree of caution.
+
+#### 7.7.2 Claim scope
+
+Some claims, like the SPIFFE ID are _explicitly_ scoped to a trust domain by their structure. Because SVIDs are always issued by a particular trust domain authority, _all_ the claims therein are _implicitly_ scoped to that trust domain.  This scoping should be preserved when presenting claims to any decision-making process.
+
+As a hypothetical example, consider a claim about the version number of the control plane software that generated the SVID.  Different trust domain control planes might use entirely different software packages, and as such the version numbers are not directly comparable in an unscoped context.
+
+
+#### 7.7.3 Interpretation of claims
+
+If claims are to be processed in making a security decision (such as whether to allow a request or connection), it is critical that the authority issuing the claim has the same meaning or interpretation of the claim as the entity making the decision.
```

Comment:
I think it is worth calling out here that there is some level of trust we place in the authority to make the assertion.

```suggestion
If claims are to be processed in making a security decision (such as whether to allow a request or connection), it is critical that the authority issuing the claim has the same meaning or interpretation of the claim as the entity making the decision, and that the validator trusts the authority to make such an assertion faithfully.
```

Reply:
Maybe I'm misunderstanding what you mean by "faithfully"

If you're referring to whether the authority makes claims that are true, then I think this is covered in the next section.